### ISSUE TITLE
UnhandledPromiseRejectionWarning on Webhook event

### ISSUE THREAD

**Issue by stjohnjohnson** [screwdriver-cd/screwdriver#422]
Getting this error when sending GitHub webhooks to beta.cd
```
(node:15) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 4): Error: Cannot wrap non-Error object
```

This is blocking the pipeline.

**Comment by minz1027** [comment]
It turned out we were reading the webhook secret for beta from the production one... Fixed! 

For the error:
```
UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 4): Error: Cannot wrap non-Error object
```

It is coming from https://github.com/screwdriver-cd/screwdriver/blob/master/features/step_definitions/git-flow.js#L147t

We did not catch error when we searched for build created from the PR event. Since the webhook was failing and no PR job/build got created so we got the error.
